app/src/main/shared/services/ModellingGroupsService.ts

```diff
@@ -1,8 +1,9 @@
-import { LocalService } from "../../shared/services/LocalService";
+import { AbstractLocalService } from "./AbstractLocalService";
 
-export class ModellingGroupsService extends LocalService {
+export class ModellingGroupsService extends AbstractLocalService {
 
     getAllGroups() {
-        return this.get("/modelling-groups/");
+        return this.setOptions({cache: 'groups'})
```

Comment:
If we ever want to invalidate the cache using this part of the key, might be a good idea to make this option and enum rather than a string, so we can keep track of the keys we're using?